### ISSUE TITLE
Generated Python header comment missing empty line beneath

### ISSUE THREAD

**Issue by B-rando1** [JacquesCarette/Drasil#3732]
I noticed it in the `ReadTable` file from GlassBR, there's no empty line between the header comment block and the comment block for the first function:

https://github.com/JacquesCarette/Drasil/blob/f34609c5fee391de84fbb112b6bbaf15b0dfffe9/code/stable/glassbr/src/python/ReadTable.py#L1-L10

My guess is that this happens whenever there are no import statements.  I took a quick look at the other targets, and it seems to only happen with the Python target.  As far as I know this is not intended.

**Comment by smiths** [comment]
Yes @B-rando1, I agree that an extra blank line would make the code easier to read.  The current behaviour was not intended.
